### PR TITLE
Update meta.py

### PR DESCRIPTION
**Pull request by russellballestrini**
Consistent quotes.  I'm going to make the same PR to the alembic docs.

I made a similar change here: https://github.com/zzzeek/alembic/pull/42

### CODE REVIEW

**Comment by stevepiercy** [comment]
This change is OK. However a separate PR should be paired to update docs in Pyramid:

* docs/quick_tour/sqla_demo/sqla_demo/models/meta.py
* docs/tutorials/wiki2/src/authentication/tutorial/models/meta.py
* docs/tutorials/wiki2/src/authorization/tutorial/models/meta.py
* docs/tutorials/wiki2/src/basiclayout/tutorial/models/meta.py
* docs/tutorials/wiki2/src/installation/tutorial/models/meta.py
* docs/tutorials/wiki2/src/models/tutorial/models/meta.py
* docs/tutorials/wiki2/src/tests/tutorial/models/meta.py
* docs/tutorials/wiki2/src/views/tutorial/models/meta.py

We also need to backport it to branches `1.9-branch` and `1.8-branch`, and make the same changes to corresponding branches for Pyramid.

**Comment by mmerickel** [comment]
Yep and don't forget that ideally PRs would be against master here instead of latest. I know it's tough because latest is the default branch.

**Comment by mmerickel** [comment]
> We also need to backport it to branches 1.9-branch and 1.8-branch, and make the same changes to corresponding branches for Pyramid.

@stevepiercy in general a contribution should only be expected to be done on master. If you really want to backport stuff that's fine, but I'm really not a fan of backporting every little change. It's just a crazy amount of work. Usually new work and improvements belong in new releases and the fact that you go above and beyond and backport it to old versions is great if you want to do it - but not something outside contributors are responsible for.

To summarize how to get this PR merged:

1) @russellballestrini please reopen it against master instead of latest. Alternatively we can just merge the branch against master manually if that doesn't happen and then close this PR manually.

2) @stevepiercy or @russellballestrini can update the pyramid documentation/tutorials to match this **on pyramid's master**.

3) @stevepiercy or some other motivated individual can backport all of this work to older branches if they so desire.

**Comment by stevepiercy** [comment]
@mmerickel sounds like a good plan. I can do both items 2 and 3 (this would be a trivial backport, unlike some monsters from the past).

**Comment by russellballestrini** [comment]
Closed in favor of PR #24 against `master`.
